tools/ss13_genchangelog.py

```diff
@@ -70,7 +70,7 @@ def dictToTuples(inp):
         with open(changelog_cache) as f:
             (_, all_changelog_entries) = yaml.load_all(f)
             failed_cache_read = False
-            
```

Comment:
# DO NOT TOUCH THE CHANGELOG GENERATOR

i repeat

# DO NOT TOUCH THE CHANGELOG GENERATOR